src/main/java/com/fileManager/fileManager/RunAfterStartUp.java

```diff
@@ -0,0 +1,35 @@
+package com.fileManager.fileManager;
+
+import com.fileManager.fileManager.model.DocumentTypeEntity;
+import com.fileManager.fileManager.repositories.DocumentTypeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Component
+public class RunAfterStartUp {
```

Comment:
No hard-coded values please, once the application s up, the user needs to set up the users/meta-data/other required data from the application itself.